service/rtc/utils.go

```diff
@@ -33,38 +34,63 @@ func generateAddrsPairs(localIPs []string, publicAddrsMap map[string]string, hos
 	var pairs []string
 	var hostOverrideIP string
 
+	// If the override is in full NAT mapping format (e.g. "EA/IA,EB/IB") we return
```

Comment:
I can't seem to find what the nat mapping format is, can you give me a link to read up on it? I'm guessing it means: external addr/internal addr,external binding/internal binding?

Reply:
Oh, letters are purely sequentials, you could have EC/IC,ED/ID and so on. These are pairs of external IP / internal IP mappings.

It's not really well documented in https://pkg.go.dev/github.com/pion/webrtc/v3#SettingEngine.SetNAT1To1IPs I had to ask to find out how to properly use it.

Reply:
I updated the config documentation slightly. Hopefully it's clearer now.